Tomes/Tome.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Newtonsoft.Json.Linq;
+
+public class Tome {
+	public delegate void OnChanged(JToken oldValue);
+	public delegate void OnDestroy();
+	public delegate void OnAdd(JToken key);
+	public delegate void OnDel(JToken key);
+
+	//
+	public static JToken Conjure(JToken data, JToken root = null) {
+		switch (data.Type) {
+		case JTokenType.Array:
+			return new TomeArray((JArray)data, root);
+		case JTokenType.Object:
+			return new TomeObject((JObject)data, root);
+		default:
+			return new TomeValue((JValue)data, root);
```

Comment:
when i checked the node-tomes repo docs and source i could not find any instance of "scalar".
Could you please point me to this so i can check?